server/utils/applications/reasonForShortNoticeDetails.ts

```diff
@@ -0,0 +1,24 @@
+import { ApprovedPremisesApplication as Application } from '../../@types/shared'
+import { retrieveOptionalQuestionResponseFromFormArtifact } from '../retrieveQuestionResponseFromFormArtifact'
+import { startDateOutsideOfNationalStandardsTimescales } from './startDateOutsideOfNationalStandardsTimescales'
+import ReasonForShortNotice from '../../form-pages/apply/reasons-for-placement/basic-information/reasonForShortNotice'
+
+export const reasonForShortNoticeDetails = (
+  application: Application,
+): {
+  reasonForShortNotice: string
+  reasonForShortNoticeOther: string
+} => {
+  const startDateOutsideOfTimescales = startDateOutsideOfNationalStandardsTimescales(application)
+  let reasonForShortNotice
+  let reasonForShortNoticeOther
+  if (startDateOutsideOfTimescales) {
```

Comment:
I don't think we need to check if the start date is outside of national timescales
`retrieveOptionalQuestionResponseFromFormArtifact` will return undefined if the start date is inside of national timescales.

Reply:
there may be scenario where first the releasedate is OutsideOfNationalStandardsTimescales and entered reasonForShortNotice and later chnaged releasedate and it is not OutsideOfNationalStandardsTimescales but still it will send reasonForShortNotice if we dont have this check startDateOutsideOfNationalStandardsTimescales

Reply:
Yes, very good point.